test/e2e/density.go

```diff
@@ -162,21 +162,18 @@ var _ = Describe("Density", func() {
 		// (metrics from other tests affects this one).
 		// TODO: Reenable once we can measure latency only from a single test.
```

Comment:
They were all being skipped. So I removed the functionality to skip them individually and skipped the whole block.
You can still unskip them by specifying them as a group or individually.